src/utils/MerkleProof.sol

```diff
@@ -37,4 +37,27 @@ library MerkleProof {
             isValid := eq(computedHash, root)
         }
     }
-}
\ No newline at end of file
+
+    function verifyCalldata(
```

Comment:
imo lets rename this to verify and the other to verifyMemory

Reply:
or actually can we just remove verifyMemory?

Reply:
Let's remove the memory version.